static/js/app.ts

```diff
@@ -6,6 +6,8 @@ import { auth } from './auth';
 
 export let theGlobalEditor: AceAjax.Editor;
 
+declare var range: any;
```

Comment:
There's not really a need to make this a global variable, since we only use it in the block below. Better to avoid making this variable unnecessarily visible (especially if it's mutable). That could only lead to mistakes later on.

Also typing it as `any` makes it possible to misuse it. We can have the compiler automatically derive the type for us if we assign the variable immediately when we declare it, that saves typing and headaches.

It would suggest making this a local, immutable value below.